### PR TITLE
add missing enum types

### PR DESCRIPTION
**Pull request by DiogoSoaress**
Export some missing enum types to be used in the frontend clients:

- [TransactionInfoType](https://safe.global/safe-client-gateway/docs/routes/transactions/models/enum.TransactionInfo.html)
- [ConflictType](https://safe.global/safe-client-gateway/docs/routes/transactions/models/summary/enum.ConflictType.html)
- [TransactionListItemType](https://safe.global/safe-client-gateway/docs/routes/transactions/models/summary/enum.TransactionListItem.html)

### CODE REVIEW

File: src/types/transactions.ts [hunk 1]
```diff
@@ -227,7 +257,7 @@ export type TransactionSummary = {
 export type Transaction = {
   transaction: TransactionSummary
   conflictType: 'None' | 'HasNext' | 'End'
```
Comment (iamacook) [inline src/types/transactions.ts:259]:
```suggestion
  conflictType: ConflictType
```

**Review by katspaugh — APPROVED** [review]
Nice 👍

**Comment by katspaugh** [comment]
You might want to bump the version too

**Comment by DiogoSoaress** [comment]
> You might want to bump the version too

bumped the patch version

**Pull request merged**